templates/admin/mooc/list_type.html.twig

```diff
@@ -0,0 +1,12 @@
+{% extends 'SonataAdminBundle:CRUD:base_list_field.html.twig' %}
+
+{% block field %}
+    {% if object.getType is defined %}
+        {% if object.getType == constant('ELEMENT_TYPE_VIDEO', object) %}
+            <span class="label label-danger">Youtube</span><br/>
+        {% endif %}
+        {% if object.getType == constant('ELEMENT_TYPE_QUIZ', object) %}
```

Comment:
`elseif` would be better as we know it's possible to get there if the 1st if condition is met.